Ladybird/WebContentView.cpp

```diff
@@ -778,12 +778,10 @@ void WebContentView::notify_server_did_unhover_link(Badge<WebContentClient>)
 
 void WebContentView::notify_server_did_click_link(Badge<WebContentClient>, AK::URL const& url, DeprecatedString const& target, unsigned int modifiers)
 {
-    // FIXME
-    (void)url;
-    (void)target;
+    if (on_open_url_in_new_tab && target == "_blank") {
+        on_open_url_in_new_tab(url, Web::HTML::ActivateTab::Yes);
```

Comment:
care to add `TODO` for choosing whether tab should be activated or not? 😅 [e.g. through a browser setting])

Reply:
I'll see about that, I got another comment and maybe the code changes.